test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_Thread.java

```diff
@@ -163,7 +163,7 @@ public void test_Constructor2() {
 			ct = new Thread(new SimpleThread(10));
 			ct.start();
 		} catch (Exception e) {
-			AssertJUnit.assertTrue("Failed to create subthread", false);
+			AssertJUnit.fail("Failed to create subthread", e);
```

Comment:
This needs to use `Assert.fail` like on line 276 below. This correction is required several other places below.

Reply:
Fixed, I hope.